pkg/ui/src/views/reports/containers/statementDiagnosticsHistory/index.tsx

```diff
@@ -97,8 +97,24 @@ class StatementDiagnosticsHistoryView extends React.Component<StatementDiagnosti
                   />
                 )}
               >
-                Download
+                Trace (.json)
               </Button>
+              <div className="crl-statements-diagnostics-view__divider" />
```

Comment:
It looks like `.crl-statements-diagnostics-view__divider` class doesn't exist and have no effect.
To add vertical divider, it is better to use following:
`<div className="crl-statements-diagnostics-view__vertical-line" />`

The same divider has to be added in next file (L109)

Reply:
That was my placeholder for figuring out the divider issue 😄 .

Latest push uses the ant Divider component.